bloomstack_core/bloomstack_core/custom/sales_invoice.json

```
@@ -265,5 +242,5 @@
    "value": "0"
   }
  ],
- "sync_on_migrate": 1
+ "sync_on_migrate": 0
```

```suggestion
 "sync_on_migrate": 1
```
we can keep it as 1 since we are deleting the naming series

---

I tried deleting naming series and sync_on_migrate to 1, it was still deleting the naming series.